aiormq/exceptions.py

```diff
@@ -141,20 +141,26 @@ class MethodNotImplemented(AMQPError):
 class DeliveryError(AMQPError):
     __slots__ = 'message', 'frame'
 
-    def __init__(self, message: DeliveredMessage, frame: spec.Frame,
-                 info: str = None):
+    reason = "Error when delivery message %r, frame %r"
```

Comment:
I suggest something like "Message is not delivered, message %r, frame %r" - to match the style of other errors.

And I don't see any value in `message` attribute here since `Basic.Nack` and `Basic.Reject` confirmation frames don't have content body with message content.  Only `Basic.Return` frame provides content body.